pinot-broker/src/test/java/org/apache/pinot/broker/requesthandler/TimeSeriesRequestHandlerTest.java

```diff
@@ -0,0 +1,151 @@
+package org.apache.pinot.broker.requesthandler;
+
+import java.util.List;
+import org.apache.pinot.spi.data.Schema;
+import org.apache.pinot.spi.exception.QueryErrorCode;
+import org.apache.pinot.spi.exception.QueryException;
+import org.apache.pinot.tsdb.spi.plan.LeafTimeSeriesPlanNode;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.*;
+
+
+public class TimeSeriesRequestHandlerTest {
+
+  private AutoCloseable _mocks;
+
+  @Mock
+  private LeafTimeSeriesPlanNode _leafNode;
+  @Mock
+  private Schema _tableSchema;
+
+  @BeforeMethod
+  public void setUp() {
+    _mocks = MockitoAnnotations.openMocks(this);
+
+    // Set up default mock behaviors
+    lenient().when(_leafNode.getTableName()).thenReturn("metrics_REALTIME");
+    lenient().when(_tableSchema.hasColumn("ts")).thenReturn(true);
+    lenient().when(_tableSchema.hasColumn("colA")).thenReturn(true);
+    lenient().when(_tableSchema.hasColumn("colB")).thenReturn(true);
+    lenient().when(_tableSchema.hasColumn("colC")).thenReturn(true);
+  }
+
+  @Test
+  void testValidateColumnNamesSuccess() {
+    when(_leafNode.getTimeColumn()).thenReturn("ts");
+    when(_leafNode.getValueExpression()).thenReturn("SUM(colA)");
```

Comment:
So value expression won't really have aggregation functions. Instead it can have transformations.

So you can test for stuff like `colA + colB`, `colA + 1`, `colA / colB`, etc.